tedge/src/cli/certificate/cli.rs

```diff
@@ -81,5 +85,9 @@ pub enum UploadCertCli {
         /// Provided username should be a Cumulocity user with tenant management permissions.
         /// The password is requested on /dev/tty, unless the $C8YPASS env var is set to the user password.
         username: String,
+
+        #[structopt(long = "no-verify-server")]
+        /// Allow upload to a server with invalid certificate, default = false.
+        verify_server_cert: bool,
```

Comment:
This variable is named in a very confusing way. `verify_server_cert` sounds like it will be true if the certificate is to be verified - but it's the opposite of that, right?

Reply:
This is `structopt`s type magic, the `bool` variable by default is false and we only want to ignore the cert check if it is provided in the cli so eg:

```shell
tedge cert upload c8y --user username
```

will result with the flag to be `false`

```shell
tedge cert upload c8y --user username --no-verify-server
```

flag is set to `true`.

Reply:
Right - so the variable should be called something like `ignore_server_certficate_validity` then, yes?

By default we want to verify the server certificate - so if you've got a variable called `verify_server_cert` set to false by default, then something is wrong.

Reply:
Thanks for coming up with a good name.